README.md

```diff
@@ -8,7 +8,7 @@ All content can be edited using just [Markdown](https://en.wikipedia.org/wiki/Ma
 
 * Front page (in English) can be modified editing [index.md](/_includes/index.md).
 * Pages can be added/edited from the [_pages](/_pages) folder, including subfolders for localized content (front page included).
-* User interface strings for all languages are in [_data/l10n](/_data/l10n/).
+* Translations are managed through [Pontoon](http://pontoon.mozilla.org/)
```

Comment:
You could add a [direct link](https://pontoon.mozilla.org/projects/activate-mozilla/), now that is live.